weekly-mission(week2)/src/features/signIn.js

```diff
@@ -0,0 +1,20 @@
+import MASTER_ACCOUNT from "../constants/accounts.js";
+import { $ } from "../constants/dom.js";
+import { addErrorMsgToIncorrectInput } from "../view/addErrorMessageToInput.js";
+
+const isMaster = (email, password) => email === MASTER_ACCOUNT.EMAIL && password === MASTER_ACCOUNT.PASSWORD;
+
+export const signInHandler = (e) => {
+  e.preventDefault();
+
+  const emailInput = $('.email').value;
```

Comment:
변수명이 조금 어색한데요~
emailInput이 되려면 이렇게 DOM 요소 자체만 가져오는 경우에 적합할 것 같아요!
`const emailInput = $('.email');`

현재는 input 태그의 value값을 가져오고 있기 때문에 email이 되는게 더 적당할 것 같아요~
```suggestion
  const email = $('.email').value;
```